src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+
+import java.awt.*;
+import java.util.*;
```

Comment:
Consider explicit import statements as suggested [here](https://se-education.org/guides/conventions/java/basic.html#package-and-import-statements).